src/middlewared/middlewared/api/v25_10_0/nfs.py

```diff
@@ -1,29 +1,57 @@
 from typing import Annotated, Literal, TypeAlias
 
-from pydantic import (
-    Field, AfterValidator
-)
+from pydantic import Field, AfterValidator
 
 from middlewared.api.base import (
-    BaseModel, Excluded, excluded_field, ForUpdateMetaclass, NonEmptyString,
+    BaseModel,
+    Excluded,
+    excluded_field,
+    ForUpdateMetaclass,
+    NonEmptyString,
     single_argument_args,
-    TcpPort, exclude_tcp_ports
+    single_argument_result,
+    TcpPort,
+    exclude_tcp_ports,
 )
 
-__all__ = ["NfsEntry",
-           "NfsUpdateArgs", "NfsUpdateResult",
-           "NfsBindipChoicesArgs", "NfsBindipChoicesResult",
-           "NfsShareEntry",
-           "NfsShareCreateArgs", "NfsShareCreateResult",
-           "NfsShareUpdateArgs", "NfsShareUpdateResult",
-           "NfsShareDeleteArgs", "NfsShareDeleteResult"]
+__all__ = [
+    "GetNFSv3ClientsEntry",
+    "GetNFSv4ClientsEntry",
+    "NfsEntry",
+    "NfsBindipChoicesArgs",
+    "NfsBindipChoicesResult",
+    "NfsClientCountArgs",
+    "NfsClientCountResult",
+    "NfsShareEntry",
+    "NfsShareCreateArgs",
+    "NfsShareCreateResult",
+    "NfsShareUpdateArgs",
+    "NfsShareUpdateResult",
+    "NfsShareDeleteArgs",
+    "NfsShareDeleteResult",
+    "NfsUpdateArgs",
+    "NfsUpdateResult",
+]
 
 MAX_NUM_NFS_NETWORKS = 42
 MAX_NUM_NFS_HOSTS = 42
 NFS_protocols = Literal["NFSV3", "NFSV4"]
 NFS_RDMA_DEFAULT_PORT = 20049
 EXCLUDED_PORTS = [NFS_RDMA_DEFAULT_PORT]
-NfsTcpPort: TypeAlias = Annotated[TcpPort | None, AfterValidator(exclude_tcp_ports(EXCLUDED_PORTS))]
+NfsTcpPort: TypeAlias = Annotated[
+    TcpPort | None, AfterValidator(exclude_tcp_ports(EXCLUDED_PORTS))
+]
+
+
+class GetNFSv3ClientsEntry(BaseModel):
+    result: list
+
+
+@single_argument_result
+class GetNFSv4ClientsEntry(BaseModel):
+    id: str
+    info: dict
+    state: dict
```

Comment:
```suggestion
    states: list[dict]
```
It looks like the old schema was wrong with this one